FinSearchUnified/Tests/PluginTest.php

```diff
@@ -226,15 +229,39 @@ public function testArticleExportWithCrossSellingCategories($crossSellingCategor
         $this->assertSame($expectedCount, $exportedCount);
     }
 
-    public function testEmptyValueNotAllowedExceptionIsThrownInExport()
+    public function exceptionProvider()
+    {
+        return [
+            'EmptyValueNotAllowedException' => [
+                'exception' =>  new EmptyValueNotAllowedException(),
```

Comment:
Add a `message` argument as well and ensure that the log message contains the given message.
```suggestion
                'exception' =>  new EmptyValueNotAllowedException('Entity not found'),
                'expectedMessage' => 'Entity not found',
```

Reply:
I can't find a way to do that, as `willThrowException` is the only exception-related assertion I could find for the mock.

Reply:
Yeah here it is kinda obvious, as you're telling the mock to throw the exception. But I want to ensure that the logged message contains the exception message.

```php
// OLD
$loggerMock->expects($this->once())
    ->method('error');

// NEW
$loggerMock->expects($this->once())
    ->method('error')
    ->with($expectedExceptionMessage);
```